dioptas/tests/widget_tests/test_NormalizedImageItem.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Dioptas - GUI program for fast processing of 2D X-ray diffraction data
+# Principal author: Clemens Prescher (clemens.prescher@gmail.com)
+# Copyright (C) 2014-2019 GSECARS, University of Chicago, USA
+# Copyright (C) 2015-2018 Institute for Geology and Mineralogy, University of Cologne, Germany
+# Copyright (C) 2019-2020 DESY, Hamburg, Germany
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Test NormalizedImageItem pyqtgraph's GraphicObject"""
+
+import numpy as np
+
+import pytest
+from pyqtgraph import GraphicsLayoutWidget
+from qtpy import QtCore
+from qtpy.QtTest import QSignalSpy, QTest
+
+from dioptas.widgets.plot_widgets.NormalizedImageItem import NormalizedImageItem
+
+NORMALIZATIONS = tuple(NormalizedImageItem._NORMALIZATIONS.keys())
+
+
+@pytest.fixture
+def normalizedImageItem(qapp, qWidgetFactory):
```

Comment:
Build on `qWidgetFactory` to provide a `NormalizedImageItem` in a displayed widget.